### ISSUE TITLE
`make test` throws "Errorf format %w has unknown verb w" error on ppc64le with latest master

### ISSUE THREAD

**Issue by sarveshtamba** [openshift/odo#3607]
I have pulled latest changes from master in my local ppc64le branch
`make bin` works fine, however `make test` throws this error which was not seen on ppc64le before the pull:-
```
# github.com/openshift/odo/pkg/devfile/adapters/kubernetes/component
pkg/devfile/adapters/kubernetes/component/adapter.go:174:10: Errorf format %w has unknown verb w
?       github.com/openshift/odo/cmd/cli-doc    [no test files]
?       github.com/openshift/odo/cmd/odo        [no test files]
ok      github.com/openshift/odo/pkg/application        (cached)
ok      github.com/openshift/odo/pkg/application/labels (cached)
?       github.com/openshift/odo/pkg/auth       [no test files]
ok      github.com/openshift/odo/pkg/catalog    (cached)
ok      github.com/openshift/odo/pkg/component  (cached)
ok      github.com/openshift/odo/pkg/component/labels   (cached)
ok      github.com/openshift/odo/pkg/config     (cached)
ok      github.com/openshift/odo/pkg/debug      (cached)
ok      github.com/openshift/odo/pkg/devfile/adapters   (cached)
ok      github.com/openshift/odo/pkg/devfile/adapters/common    (cached)
?       github.com/openshift/odo/pkg/devfile/adapters/docker    [no test files]
ok      github.com/openshift/odo/pkg/devfile/adapters/docker/component  (cached)
ok      github.com/openshift/odo/pkg/devfile/adapters/docker/storage    (cached)
ok      github.com/openshift/odo/pkg/devfile/adapters/docker/utils      (cached)
?       github.com/openshift/odo/pkg/devfile/adapters/kubernetes        [no test files]
FAIL    github.com/openshift/odo/pkg/devfile/adapters/kubernetes/component [build failed]
ok      github.com/openshift/odo/pkg/devfile/adapters/kubernetes/storage        (cached)
ok      github.com/openshift/odo/pkg/devfile/adapters/kubernetes/utils  (cached)
ok      github.com/openshift/odo/pkg/devfile/parser     (cached)
ok      github.com/openshift/odo/pkg/devfile/parser/context     (cached)
ok      github.com/openshift/odo/pkg/devfile/parser/data        (cached)
?       github.com/openshift/odo/pkg/devfile/parser/data/1.0.0  [no test files]
?       github.com/openshift/odo/pkg/devfile/parser/data/2.0.0  [no test files]
?       github.com/openshift/odo/pkg/devfile/parser/data/common [no test files]
ok      github.com/openshift/odo/pkg/devfile/validate   (cached)
ok      github.com/openshift/odo/pkg/envinfo    (cached)
?       github.com/openshift/odo/pkg/exec       [no test files]
ok      github.com/openshift/odo/pkg/kclient    (cached)
?       github.com/openshift/odo/pkg/kclient/fake       [no test files]
ok      github.com/openshift/odo/pkg/lclient    (cached)
?       github.com/openshift/odo/pkg/log        [no test files]
?       github.com/openshift/odo/pkg/log/fidget [no test files]
?       github.com/openshift/odo/pkg/machineoutput      [no test files]
ok      github.com/openshift/odo/pkg/notify     (cached) [no tests to run]
ok      github.com/openshift/odo/pkg/occlient   (cached)
?       github.com/openshift/odo/pkg/odo/cli    [no test files]
?       github.com/openshift/odo/pkg/odo/cli/application        [no test files]
?       github.com/openshift/odo/pkg/odo/cli/catalog    [no test files]
?       github.com/openshift/odo/pkg/odo/cli/catalog/describe   [no test files]
?       github.com/openshift/odo/pkg/odo/cli/catalog/list       [no test files]
?       github.com/openshift/odo/pkg/odo/cli/catalog/search     [no test files]
ok      github.com/openshift/odo/pkg/odo/cli/catalog/util       (cached)
?       github.com/openshift/odo/pkg/odo/cli/component  [no test files]
?       github.com/openshift/odo/pkg/odo/cli/component/ui       [no test files]
?       github.com/openshift/odo/pkg/odo/cli/config     [no test files]
?       github.com/openshift/odo/pkg/odo/cli/debug      [no test files]
?       github.com/openshift/odo/pkg/odo/cli/login      [no test files]
?       github.com/openshift/odo/pkg/odo/cli/logout     [no test files]
?       github.com/openshift/odo/pkg/odo/cli/plugins    [no test files]
?       github.com/openshift/odo/pkg/odo/cli/preference [no test files]
?       github.com/openshift/odo/pkg/odo/cli/project    [no test files]
?       github.com/openshift/odo/pkg/odo/cli/registry   [no test files]
ok      github.com/openshift/odo/pkg/odo/cli/registry/util      (cached)
ok      github.com/openshift/odo/pkg/odo/cli/service    (cached)
ok      github.com/openshift/odo/pkg/odo/cli/service/ui (cached)
ok      github.com/openshift/odo/pkg/odo/cli/storage    (cached)
?       github.com/openshift/odo/pkg/odo/cli/ui [no test files]
?       github.com/openshift/odo/pkg/odo/cli/url        [no test files]
ok      github.com/openshift/odo/pkg/odo/cli/utils      (cached)
?       github.com/openshift/odo/pkg/odo/cli/version    [no test files]
?       github.com/openshift/odo/pkg/odo/genericclioptions      [no test files]
ok      github.com/openshift/odo/pkg/odo/util   (cached)
ok      github.com/openshift/odo/pkg/odo/util/completion        (cached)
ok      github.com/openshift/odo/pkg/odo/util/experimental      (cached)
ok      github.com/openshift/odo/pkg/odo/util/pushtarget        (cached)
ok      github.com/openshift/odo/pkg/odo/util/validation        (cached)
ok      github.com/openshift/odo/pkg/preference (cached)
ok      github.com/openshift/odo/pkg/project    (cached)
ok      github.com/openshift/odo/pkg/secret     (cached)
ok      github.com/openshift/odo/pkg/service    (cached)
ok      github.com/openshift/odo/pkg/storage    (cached)
ok      github.com/openshift/odo/pkg/storage/labels     (cached)
ok      github.com/openshift/odo/pkg/sync       (cached)
ok      github.com/openshift/odo/pkg/testingutil        (cached)
?       github.com/openshift/odo/pkg/testingutil/filesystem     [no test files]
ok      github.com/openshift/odo/pkg/url        (cached)
ok      github.com/openshift/odo/pkg/url/labels (cached)
ok      github.com/openshift/odo/pkg/util       (cached)
?       github.com/openshift/odo/pkg/version    [no test files]
ok      github.com/openshift/odo/pkg/watch      (cached)
make: *** [Makefile:144: test] Error 2
[root@shivani2-bastion odo]#
```

**Comment by sarveshtamba** [comment]
Ran `UNIT_TEST_ARGS=-v make test` and provided the specific failure log output below - 
[make-test-failures.txt](https://github.com/openshift/odo/files/4946988/make-test-failures.txt)

**Comment by amitkrout** [comment]
@sarveshtamba Which go version you are using ?

**Comment by sarveshtamba** [comment]
> @sarveshtamba Which go version you are using ?

@amitkrout 
```
[root@shivani2-cdeb-bastion odo]# go version
go version go1.12.13 linux/ppc64le
```

**Comment by amitkrout** [comment]
> > @sarveshtamba Which go version you are using ?
> 
> @amitkrout
> 
> ```
> [root@shivani2-cdeb-bastion odo]# go version
> go version go1.12.13 linux/ppc64le
> ```

@sarveshtamba In our CI we are using go version 1.13.x (https://github.com/openshift/odo/blob/master/.travis.yml#L12) infact our local dev environment runs on 1.13.* or more. Can you please update your go version to 1.13.* or more and give make test a try.

**Comment by sarveshtamba** [comment]
@amitkrout tried with go v1.14.6 However facing the below issue with `make goget-tools` itself.
```
[root@shivani2-bastion odo]# make goget-tools
go get -u github.com/Masterminds/glide
go: github.com/Masterminds/glide upgrade => v0.13.3
go: finding module for package github.com/codegangsta/cli
go: finding module for package github.com/Masterminds/semver
go: finding module for package github.com/Masterminds/vcs
go: finding module for package gopkg.in/yaml.v2
go: finding module for package github.com/mitchellh/go-homedir
go: found github.com/codegangsta/cli in github.com/codegangsta/cli v1.22.4
go: found github.com/Masterminds/semver in github.com/Masterminds/semver v1.5.0
go: found github.com/Masterminds/vcs in github.com/Masterminds/vcs v1.13.1
go: found github.com/mitchellh/go-homedir in github.com/mitchellh/go-homedir v1.1.0
go: found gopkg.in/yaml.v2 in gopkg.in/yaml.v2 v2.3.0
go: github.com/Masterminds/glide imports
        github.com/codegangsta/cli: github.com/codegangsta/cli@v1.22.4: parsing go.mod:
        module declares its path as: github.com/urfave/cli
                but was required as: github.com/codegangsta/cli
make: *** [Makefile:104: goget-tools] Error 1
[root@shivani2-bastion odo]# go version
go version go1.14.6 linux/ppc64le
```

**Comment by sarveshtamba** [comment]
@amitkrout any more inputs on this one?

**Comment by girishramnani** [comment]
we are switching to `go mod` once this is https://github.com/openshift/odo/pull/3866 merged

**Comment by sarveshtamba** [comment]
> we are switching to `go mod` once this is #3866 merged

Sure, will hold on until #3866 is merged. Thanks for the input.

**Comment by sarveshtamba** [comment]
Resolved with #3891 , closing this for now.
